blankiety_galantow/resources/game/index.html

```diff
@@ -74,6 +91,24 @@ <h3>Ustawienia</h3>
                                    v-model="newSettings.open">
                             <label for="is-open">Zezwól na dołączenie nowych graczy</label>
                         </p>
+                        <p  v-if="me.admin">
+                            <input 
+                                id="is-password-protected" 
+                                name="is-password-protected" 
+                                class="w3-check" 
+                                type="checkbox"
+                                @change="changePasswordRequired($event)">
+                            <label for="is-password-protected">Zabezpieczenie hasłem</label>
+                            <p v-if="settingsPasswordRequired" >
+                                Hasło: <input type="text" minlength="5" v-model="newSettings.password">
+                            </p>
+                            <div v-if="6 > newSettings.password.length && settingsPasswordRequired" style="text-align: center;">
```

Comment:
Zlituj się nad czytającymi kod xD
```suggestion
                            <div v-if="newSettings.password.length < 6 && settingsPasswordRequired" style="text-align: center;">
```